README.md

```diff
@@ -15,115 +15,24 @@ Rust ZeroMQ bindings.
 # About
 
 The `zmq` crate provides bindings for the `libzmq` library from the
-[ZeroMQ](https://zeromq.org/) project. The API exposed by `zmq` should
-be safe (in the usual Rust sense), but it follows the C API closely,
-so it is not very idiomatic. Also, support for `libzmq` API in "draft"
-state is considered out-of-scope for this crate; this includes
-currently, as of libzmq 4.3.3:
+[ZeroMQ](https://zeromq.org/) project. This project is a fork of the 
```

Comment:
The README changes are changes related to the forked project rather than this project. I am unsure whether the compatibility/installation section is still needed, since the `cmake` dependency was removed, but if it is it'd be nice to have that section back/updated.